### PR TITLE
docs: fix glossary unclosed div

### PR DESCRIPTION
**Pull request by kapunahelewong**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Most of the glossary is at the mercy of an unclosed callout.
Issue Number: N/A

## What is the new behavior?

Add closing div.

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information
@petebacondarwin Thanks for spotting this!

### CODE REVIEW

**Comment by ocombe** [comment]
There's another one line ~632 just after `## Router module`

**Comment by ocombe** [comment]
PS: I don't think that's your supposed to close these `<div class="l-sub-section">` divs, but remove them instead, they make no sense here

**Comment by kapunahelewong** [comment]
@ocombe Thank you for double checking me. Fixed.

File: aio/content/guide/glossary.md [hunk 1]
```diff
@@ -511,10 +509,6 @@ You rarely access Angular feature modules directly. You usually import them from
 
 ## NgModule
 
-<div class="l-sub-section">
-
-
-
 Helps you organize an application into cohesive blocks of functionality.
 An NgModule identifies the components, directives, and pipes that the application uses along with the list of external NgModules that the application needs, such as `FormsModule`.
```
Comment (ocombe) [inline aio/content/guide/glossary.md]: you missed the closing div (line 527) on this one

Comment (kapunahelewong) [reply aio/content/guide/glossary.md]: Got it. Thank you!

**Comment by kapunahelewong** [comment]
@ocombe would you review this again when you have a moment? I've removed the subsection (and the dangling bit) because it was the entire definition of bootstrapping, as opposed to a note in addition to a definition.

**Review by ocombe — APPROVED** [review]

**Comment by ocombe** [comment]
Sounds good to me!

**Pull request closed without merging**